ttax/ops.py

```diff
@@ -167,6 +167,30 @@ def flat_inner(a, b):
   return res
 
 
+def tt_matvec(A, x):
```

Comment:
Again, as discussed above I would prefer to instead just have an explicit convertion between a tensor and a vector-matrix